python/tests/ecl/test_sum.py

```diff
@@ -287,23 +288,32 @@ def test_invalid(self):
 
 
     def test_kw_vector(self):
-        case = createEclSum("CSV" , [("FOPT", None , 0) , ("FOPR" , None , 0), ("FGPT" , None , 0)],
-                            sim_length_days = 100,
-                            num_report_step = 10,
-                            num_mini_step = 10,
-                            func_table = {"FOPT" : fopt,
-                                          "FOPR" : fopr ,
-                                          "FGPT" : fgpt })
-        kw_list = EclSumKeyWordVector( case )
+        case1 = createEclSum("CSV" , [("FOPT", None , 0) , ("FOPR" , None , 0), ("FGPT" , None , 0)],
+                             sim_length_days = 100,
+                             num_report_step = 10,
+                             num_mini_step = 10,
+                             func_table = {"FOPT" : fopt,
+                                           "FOPR" : fopr ,
+                                           "FGPT" : fgpt })
+
+        case2 = createEclSum("CSV" , [("FOPR", None , 0) , ("FOPT" , None , 0), ("FWPT" , None , 0)],
+                             sim_length_days = 100,
+                             num_report_step = 10,
+                             num_mini_step = 10,
+                             func_table = {"FOPT" : fopt,
+                                           "FOPR" : fopr ,
+                                           "FWPT" : fgpt })
+
+        kw_list = EclSumKeyWordVector( case1 )
         kw_list.add_keyword("FOPT")
-        kw_list.add_keyword("FOPR")
         kw_list.add_keyword("FGPT")
+        kw_list.add_keyword("FOPR")
 
-        t = case.getDataStartTime( ) + datetime.timedelta( days = 43 );
-        data = case.get_interp_row( kw_list , t )
-        for d1,d2 in zip(data, [ case.get_interp("FOPT", date = t),
-                                 case.get_interp("FOPT", date = t),
-                                 case.get_interp("FOPT", date = t) ]):
+        t = case1.getDataStartTime( ) + datetime.timedelta( days = 43 );
+        data = case1.get_interp_row( kw_list , t )
+        for d1,d2 in zip(data, [ case1.get_interp("FOPT", date = t),
+                                 case1.get_interp("FOPT", date = t),
+                                 case1.get_interp("FOPT", date = t) ]):
```

Comment:
I have a question regarding `get_interp_row`, but it doesn't really belong to this PR.  But that function in `c` seems to link `ecl_sum` back to `ecl_sum_vector`.  Could it not be placed in `ecl_sum_vector` instead?

Reply:
I have not looked at this code right now - but I *think* you are mxing two different types of vectors:

`ecl_sum_vector`: is one keyword, all timesteps.
`ecl_sum_keyword_vector`: A vector/collection of keywords, which are typically used to evaluate many keywords at the sime time point [which has a significant startup cost].